src/components/OpenLoan/TokenUserInput.tsx

```diff
@@ -0,0 +1,78 @@
+import Image from 'next/image'
+import { Token } from '../Tokens'
+
+const style = {
+  tokenUserInputContainer: 'flex justify-around mt-2',
+  tokenAmountContainer: '',
+  loanAmountInput: 'bg-transparent placeholder:text-neutrals-600 outline-none w-full text-2xl text-neutrals-200',
+  maxButton: 'w-[2rem] text-center bg-neutrals-900 bg-opacity-60 drop-shadow-lg cursor-pointer hover:bg-opacity-70 text-xxs font-normal p-0.5 rounded-sm text-accents-royalBlue text-opacity-50 hover:text-opacity-80 mt-1',
+  tokenPairContainer: 'w-[12rem]',
+  tokenPairContent: 'flex justify-end space-x-3',
+  tokenPairIcons: 'relative w-[2rem] h-[2rem] self-center',
+  tokenAIcon: 'mt-0.5',
+  tokenBIcon: 'absolute top-0.5 -right-1 -z-10',
+  tokenPairSymbol: 'text-lg',
+  tokenPairBalance: 'font-semibold text-xxs text-neutrals-600 tracking-wide text-right mt-2',
+}
+
+interface TokenUserInputProps {
+  collateralType: string
+  token0: Token
+  token1: Token
+}
+
+export const TokenUserInput = ({ collateralType, token0, token1 }: TokenUserInputProps) => {
```

Comment:
Is this component reused? If not, possibly rename to CollateralUserInput.

Reply:
I was thinking of making it reusable and just passing the tokens and a type (whether with one token or both). However, that would increase the complexity of how to add support for the other component so for now I'll keep it for collateral type use.

Reply:
done